### PR TITLE
GFORMS-2013: Update 15 minute timeout counter periodically when user …

### PR DESCRIPTION
**Pull request by lxol**
…is active on the keyboard

### CODE REVIEW

File: public/javascripts/gformSessionTimeout.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};
+  var lang = (global.gform && global.gform.lang) || "en";
+
+  function GformSessionTimeout() {
+    var self = this;
+
+    function init() {
+      $("input").keypress(broadcastTimestamp);
```
Comment (josef-vlach) [inline public/javascripts/gformSessionTimeout.js:12]: Can we throttle this and broadcast only every 5 minutes or so?

File: public/javascripts/gformSessionTimeout.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+(function(global) {
+  "use strict";
+
+  var $ = global.jQuery;
+  var GOVUK = global.GOVUK || {};
+  var lang = (global.gform && global.gform.lang) || "en";
```
Comment (josef-vlach) [inline public/javascripts/gformSessionTimeout.js:6]: Not needed

**Review by josef-vlach — CHANGES_REQUESTED** [review]

File: public/javascripts/gformSessionTimeout.js (hunk 1 above)
Comment (lxol) [reply public/javascripts/gformSessionTimeout.js:9]: done

File: public/javascripts/gformSessionTimeout.js (hunk 2 above)
Comment (lxol) [reply public/javascripts/gformSessionTimeout.js]: removed

**Review by josef-vlach — APPROVED** [review]

**Pull request merged**